Sources/Memberlist.php

```diff
@@ -327,13 +327,17 @@ function MLAll()
 		$limit = $second_offset - ($memberlist_cache['num_members'] - $_REQUEST['start']) - ($second_offset > $memberlist_cache['num_members'] ? $cache_step_size - ($memberlist_cache['num_members'] % $cache_step_size) : 0);
 	}
 
+	$custom_fields_qry = '';
+	if (!empty($context['custom_profile_fields']['join'][$col]))
+		$custom_fields_qry = $context['custom_profile_fields']['join'][$col];
```

Comment:
I think you might want to use `$_REQUEST['sort']` instead of `$col` here. `$col` will be set to the key of whatever the last element of `$context['columns']` happens to be, and I don't think that is what we want.